artacle-swagger.yaml

```diff
@@ -825,6 +827,641 @@ paths:
+                          example: 0.0414
+                        topTraitValuationUSD:
+                          type: number
+                          description: Token trait valuation in USD
+                          format: float
+                          example: 0.0414
+                        rarityReward:
+                          type: number
+                          description: Token rarity reward
+                          format: float
+                          example: 0.0414
+                        rarityRewardUSD:
+                          type: number
+                          description: Token rarity reward in USD
+                          format: float
+                          example: 0.0414
+                        isSpecialCase:
+                          type: number
+                          enum:
+                            - 0
+                            - 1
+                          description: Indicates if tokens valuation is computed in exceptional/special way due to token market specifics
+                          example: 0
+                        ARank:
+                          type: number
+                          description: Token A-Rank
+                          format: uint32
+                          example: 4476
+                        rarityRank:
+                          type: number
+                          description: Token ratiry rank
+                          format: uint32
+                          example: 2056
+                        valuationTimestamp:
+                          type: number
+                          format: uint64
+                          description: Unix timestamp when metrics are evaluated
+                          example: 1705667581
+                        mintPrice:
+                          type: number
+                          description: Token mint price
+                          format: float
+                          example: 0.0414
+                        mintPriceUSD:
+                          type: number
+                          description: Token mint price in USD
+                          format: float
+                          example: 102.768876
+                        listingPrice:
+                          type: number
+                          description: Token current listing price
+                          format: float
+                          example: 0.0414
+                        listingPriceUSD:
+                          type: number
+                          description: Token current listing price in USD
+                          format: float
+                          example: 102.768876
+                        listingTimestamp:
+                          type: number
+                          format: uint64
+                          description: Unix timestamp when token is listed
+                          example: 1705667581
+                        isListed:
```

Comment:
Now I'm thinking it is excessive. See, listingPrice=null indicates it is not listed. Should we get rid of it?